guava/src/test/java/org/jdbi/v3/guava/TestGuavaCollectors.java

```diff
@@ -99,21 +98,21 @@ public void immutableSortedSetWithComparator() {
 
         assertThat(set).containsExactlyElementsOf(expected.stream()
                 .sorted(comparator)
-                .collect(toList()));
+                .collect(Collectors.toList()));
     }
 
     @Test
     public void optionalPresent() {
         Optional<Integer> shouldBePresent = h2Extension.getSharedHandle().createQuery("select intValue from something where intValue = 1")
             .collectInto(new GenericType<Optional<Integer>>() {});
-        assertThat(shouldBePresent).contains(1);
+        assertThat(shouldBePresent.get()).isOne();
```

Comment:
this change is necessary because now, the guava assert no longer is present. We use this library specifically because it tests the guava types. Why is that removal useful? It is part of the test code, so it does not increase the footprint of what we ship but make our tests less specific

Reply:
changed it back